src/login/logind.c

```diff
@@ -323,7 +323,9 @@ static int manager_enumerate_seats(Manager *m) {
 
                 s = hashmap_get(m->seats, de->d_name);
                 if (!s) {
-                        unlinkat(dirfd(d), de->d_name, 0);
+                        if (unlinkat(dirfd(d), de->d_name, 0) < 0)
+                                log_warning("Failed to remove /run/systemd/seats/%s: %m",
+                                            de->d_name);
```

Comment:
everybody's favourite task: adding error handling.

We've got a different approach for each of the three calls in this commit... and each call _looks_ like it's filling the exact same role.

The journald one sounds a bit worrying, if it could later restore a stale stream with the wrong fd?  Should the journald one be raised to LOG_WARNING as well?

I'm not really worried about the udev one being ignored.

Reply:
I made them both `log_warning` now.